models/post_tag_vote.go

```diff
@@ -31,14 +31,17 @@ func (v *PostTagVote) FindByUK(postID int, tagID int, userID int) error {
 	return nil
 }
 
+// DeleteByUK - delete a PostTagVote in the database by unique keys
+func (v *PostTagVote) DeleteByUKWithTx(tx Transaction, postID int, tagID int, userID int) error {
+	_, err := tx.Exec("delete from posts_tags_votes where post_id = ? and tag_id = ? and voter_id = ?", postID, tagID, userID)
+	return err
+}
+
 // CreatePostTagVote - create the PostTagVote with post and tag information
 func (v *PostTagVote) CreatePostTagVote() error {
 	// create a new PostTag association
 	_, err := DBConn.Exec("INSERT INTO posts_tags_votes (post_id, tag_id, voter_id) VALUES (?, ?, ?)", v.PostID, v.TagID, v.VoterID)
-	if err != nil {
-		return err
-	}
-	return nil
+	return err
```

Comment:
this is more concise. personally i love seeing "return nil" at the end of a func that returns an error, but mostly just so that I can know that by the time we get to the end of it all the error is nothing. mostly just a headspace thing that I have that i think is worth the extra if statement, but I'm down to keep things nice and short.